src/node_buffer.h

```diff
@@ -29,7 +29,9 @@ namespace node {
 
 namespace Buffer {
 
-static const size_t kMaxLength = v8::TypedArray::kMaxLength;
+static constexpr size_t kMaxLength =
+    v8::TypedArray::kMaxLength < 0x10000000000ull ? v8::Uint8Array::kMaxLength
```

Comment:
Might be better to use std::min here

Reply:
Can you make a concrete suggestion? I know little about C++ features/syntax

Reply:
Definitely:

```
static const size_t kMaxLength = std::min(v8::TypedArray::kMaxLength, 0x10000000000ull);
```

Reply:
`#include <algorithm>` :-)

But I didn't suggest std::min precisely for that reason: it pulls in another header file and node_buffer.h is sort of public (included by add-ons.)